Title: Dharsy D. Rodriguez Velazquez
Date: 2023-10-17
Slug: dharsy
Category: People
Tags: bioinformatics

# Dharsy D. Rodriguez Velazquez
## Contact info:
e-mail

  - [dharsy.rodriguez@upr.edu]

LinkedIn

 - www.linkedin.com/in/dharsy-rodriguez

## Bio

Social Sciences student-researcher at Computer Sciences Department. IDGeNe student program at the University of Puerto Rico, Rio Piedras Campus. During the time I will be working with my peers and the Profesor in Bioinformatic of gene expression.

## Research Goals

To combine social sciences, public health (Geentics Epidemiology) with cutting-edge biological research.

## Research Abstract

During the expression of genes in tissues there is cell heterogeneity which causes a massive amount of voluminous information about the normal cells and anomalous cells. Bioinformatics is a field that can provide tools to analyze large data sets. In bioinformatics, computer technology is the main protagonist, it helps to analyze and translate biological data.  Hence, we will be using this field to evaluate the differential expression in cancer tissues of humans, using publicly available datasets. We selected samples of homo sapiens colon colorectal tumor core, and adjacent non-malignant colon tissue to study. Our goal is to identify gene patterns, with this in mind the method that will be used to process the data is Binarization, especially the Binary Differential Analysis (BDA). The Binary Differential Analysis has the advantage of reproducibility of data, it reduces the amount of information and detects biologically relevant genes. BDA collects the genes that were expressed as a non-zero value which was assigned a one, if the gene did not express it will be a zero value. 

1. Gerard A. Bouland, Ahmed Mahfouz, Marcel J.T. Reinders. The rise of sparser single-cell RNAseq datasets; consequences and opportunities. BioRxiv 2022.05.20.492823; doi: [https://doi.org/10.1101/2022.05.20.492823](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdoi.org%2F10.1101%2F2022.05.20.492823&data=05%7C01%7Cdharsy.rodriguez%40upr.edu%7Ce570fea961614a82d0b308db669ba645%7C0dfa5dc0036f461599e494af822f2b84%7C0%7C0%7C638216590908196365%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=n6MmgeelVeeLroGdKVcsV%2FuhBdXFpTxBYkp5RRhNvPQ%3D&reserved=0)

1. Satija, R., Farrell, J.A., Gennert, D., Schier, A.F. and Regev, A. (2015) Spatial reconstruction of single-cell gene expression data. Nat. Biotechnol. Retrieved from: https://doi.org/10.1038/n
bt.3192

1. He D, Zakeri M, Sarkar H, Soneson C, Srivastava A, Patro R. (2022) Alevin-fry unlocks rapid, accurate and memory-frugal quantification of single-cell RNA-seq data. Nat Methods. Mar;19(3):316-322. doi: 10.1038/s41592-022-01408-3. Epub 2022 Mar 11. PMID: 35277707; PMCID: PMC8933848.

## Weekly UPDATES
June 5-9 2023
IDGeNe first meeting

### July 24-28,2023

### July 31-August 4,2023

### August 14-18,2023

### August 21-25,2023

### August 28-1 September,2023

### September 4-8,2023

### September 11-15,2023

### September 18-22,2023

### September 25-29,2023

### October 2-6,2023

### October 9-13,2023

### October 16-20,2023

### October 23-27,2023

### October 30-3 November,2023

### November 6-10,2023

### November 13-17,2023

### November 20-24,2023

### November 27-1 December,2023

### December 4-8,2023, 2023

### December 11-15,2023, 2023

### December 18-22,2023, 2023
 -Christmas Break
### December 25-31, 2023
 -MIT prework, Survey and documents

### January 1,2023
 -Introduction and expectations with Mandana Sassanfar
  -Magdalena Krubner "Review of python basics"
  -Magdalena Krubner "Simple coding with python"

### January 2,2023: Python Applications/Machine Learning 
 -Review and discussion with Mandana Sassanfar
 -Hannah JObs "PLotting and Curve fitting using python"
 -Lunch with faculty: Lindsay Case Kristen Knouse, Sinisa Hvartin, Mehrdad Jazayeri, Francisco Sanche, and Brandon Weissbourd.
 -Taylor Baum"Machine Learning applications to studying the brain"





### January 3,2023:Cognitive Neuroscince/Signal Processing
 -Federico Claudi "Brain Computer Interfaces-theory&Practice"
 -Lunch with faculty: Adam Martin, David Page, Eliezer Calo, Ed Boyden, Chris Burge, Matt Vander Heiden.
 -Taylor Baum "Signal processing applied to neuroscience"

## January 4,2023: CryoEM
 -Review and discussion with Mandana Sassanfar
 -Mary Ellen Wiltrout "MIT digital learning and on-line courses"
 -Lecture: Professor Joey Davis, "Introduction to CryoEM"
 -Lunch with Faculty: Joey Davis, Cathy Drennan, Becky Lamson, Amy Keating, Mark Harnett, Erin Chen
 -Joey Davis and lab members "CryoEM Data processing and analysis"

### January 5,2023:Cognitive neuroscience/ Lab Tours
 -Review and discussion with Mandana Sassanfar
 -Lecure: Professor Nancy Kanwisher, "Functional Imaging of the Human Brain"
 -Lunch with faculty: Steve Flavell, Nancy Kanwisher, Robert Desimone. 3 concurrent Tours, fMRI live demo with Steven Shannon, Athinoula, A. Martinos Imaging Center OR, Tour of MEG facilities with Dimitri Pantazi, CryoEM tour with Sarah Sterling.
 -Bryan Medina "What makes music so memorable"


### January 6,2023:GENOMICS/GWAS/RNAseq
 -Review and discussion with Mandana Sassanfar
 -Lecture: Profesor Olivia Corradin, "Human genetics and genomic studies of disease"
 -Lunh with grad students and faculty: Olivia Corradin and Seychelle Vos,Giselle Valdes & Hannah Jacob "RNAseq Data analysis"



### January 6,2023:GENOMICS/GWAS/RNAseq
 -Review and discussion with Mandana Sassanfar
 -Lecture: Profesor Olivia Corradin, "Human genetics and genomic studies of disease"
 -Lunh with grad students and faculty: Olivia Corradin and Seychelle Vos,Giselle Valdes & Hannah Jacob "RNAseq Data analysis"
 
### January 7,2023:GENOMICS/GWAS/RNAseq/Information Sessions
 -Kamal Maher, "Spatial gene expression patterns using python"
 -Grad students/Postbacs panel

